### PR TITLE
Removed Bootstrap class and migrated Navbar to Tailwind CSS

### LINKED ISSUE #1412: Feature Request: Replace Bootstrap with TailwindCSS for Navbar
**Is your feature request related to a problem? Please describe.**
The Navbar currently uses Bootstrap and Tailwind css partially, which is not ideal as the site is moving towards TailwindCSS.

**Describe the solution you'd like**
Replace Bootstrap with Tailwind CSS for navbar.

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by thedudeontitan**
Replacing Bootstrap classes with TailwindCSS Fixes: #1412

Navbar made with TailwindCSS:

image

image

### CODE REVIEW

**Review by DonnieBLT — CHANGES_REQUESTED** [review]
just two np

File: website/templates/includes/navbar.html [hunk 1]
```diff
@@ -65,87 +102,90 @@
             <div class="items-center w-full h-full nav-hidden lg:!block">
                 <i class="fa fa-search relative left-[28px] top-[-2px] text-gray-700 text-base nav-hidden lg:!inline-block"></i>
                 <input type="text" name="query" placeholder="{% trans "Search" %}" class="w-[200px] h-[38px] pl-[30px] nav-hidden lg:!inline-block bg-[#ececece1] outline-[#e4e4e4] outline-[1px]">
-                <a href="/report/" class="nav-hidden lg:!inline-block bg-red-500 hover:bg-red-600 h-[38px] py-[8px] px-[25px] whitespace-nowrap text-2xl font-bold text-white leading-[21px] cursor-pointer hover:text-white hover:no-underline">
-                    {% trans "Report a Bug" %}
+                <a href="/report/" class="lg:!inline-block bg-red-500 hover:bg-red-600 h-[38px] py-[8px] px-[25px] whitespace-nowrap !text-2xl font-bold text-white leading-[21px] cursor-pointer hover:text-white hover:no-underline">
+                    Report a Bug
```
Comment (DonnieBLT) [inline website/templates/includes/navbar.html:106]: please make sure to add the trans tags

**Review by DonnieBLT — CHANGES_REQUESTED** [review]
needs the trans tags

**Comment by thedudeontitan** [comment]
Added the trans tags

**Comment by DonnieBLT** [comment]
I am not sure why there was added javascript, can you please take another look at this?  I think the task may have been solved with just tailwind and no needed js.  Feel free to reopen the PR with the changes.  Thank you.

**Pull request closed without merging**